Unix ncurses dev lib required

**Describe the bug**
ncurses dev lib is required by a dependency. This is not documented nor listed in the `unix_env_setup.sh` script.

Error when this dependency is missing
```
CMake Error at cmake/readline.cmake:71 (MESSAGE):
  Curses library not found.  Please install appropriate package,

      remove CMakeCache.txt and rerun cmake.On Debian/Ubuntu, package name is libncurses5-dev, on Redhat and derivates it is ncurses-devel.
Call Stack (most recent call first):
  cmake/readline.cmake:100 (FIND_CURSES)
  cmake/readline.cmake:191 (MYSQL_USE_BUNDLED_EDITLINE)
  CMakeLists.txt:781 (MYSQL_CHECK_EDITLINE)
```

--- a/unix_env_setup.sh
+++ b/unix_env_setup.sh
@@ -2,7 +2,7 @@
 
 sudo apt-add-repository -y "ppa:ubuntu-toolchain-r/test"
 sudo apt-get update
-sudo apt-get install g++-7 unzip autoconf libtool libssl-dev python perl
+sudo apt-get install g++-7 unzip autoconf libtool libssl-dev python perl libncurses5-dev libreadline-dev
 
 git submodule update --init --recursive
 

